### PR TITLE
Fix Border on-hover issue for sets not yet in Standard

### PR DESCRIPTION
**Pull request by troyfitzwater**
This fixes  [issue #156](https://github.com/glacials/whatsinstandard/issues/156) by giving all sets the faded-out border that sets currently in Standard have, and giving a thicker and darker border on-hover for every set to keep that sensation of "about to click a button." 

border_on_released_set

Illegal_Set_Border_After_Hover

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -89,7 +89,8 @@ <h1>What's in Standard?</h1>
             <div v-for="round in truncate(rounds(undropped(sets)))" class="col mb-4">
               <div class="row row-cols-1 row-cols-sm-2 border mx-0 py-3 h-100">
                 <div class="col my-3" v-for="set in round">
-                  <a class="text-center p-3 mb-4 card text-dark h-100 justify-content-between" :class="{'bg-light': isReleased(set), 'text-muted border-0': !isReleased(set)}" style="min-height: 5rem"
+                  <!-- ja;lksdjfl;kasdjfljasdlfjl;sda -->
```
Comment (glacials) [inline index.html:92]: ?

File: css/style.css [hunk 2]
```diff
@@ -187,4 +191,4 @@ a.low-distraction:hover {
   bottom: -12px;
   right: 5px;
   background-color: white;
-}
+}
```
Comment (glacials) [inline css/style.css:194]: Newline please :)

File: index.html (hunk 1 above)
Comment (troyfitzwater) [reply index.html:92]: Oh, that I forgot to get rid of, I think I just needed to "bookmark" something to see when quickly scrolling by. Sorry lol

File: css/style.css (hunk 2 above)
Comment (troyfitzwater) [reply css/style.css]: Whoops, missed that. Added!

**Comment by troyfitzwater** [comment]
Hey there, thanks for looking this over. I just eased back the border to 1px to fix that shifting issue. Also took care of those other reviews you mentioned. Let me know if that looks/feels better to you.

**Comment by tooomm** [comment]
That looks really cool, thanks for the PR and quick fix @troyfitzwater 👍 

However, I realized an issue with it: #165

**Pull request merged**